torchtune/dev/rl/workers/trainers/pytorch_actor.py

```diff
@@ -739,10 +739,10 @@ def _prepare_trajectory(self, raw_trajectory):
         seq_lens = raw_trajectory.seq_lens
         answers = raw_trajectory.answers
         policy_version = raw_trajectory.policy_version
-        rewards = raw_trajectory.rewards
+        # rewards = raw_trajectory.rewards
```

Comment:
This one we should keep: it's just the total reward

Reply:
If we don't do monte carlo advantage estimation, then these are going to be used